ods_ci/tests/Resources/Page/Operators/ISVs.resource

```diff
@@ -37,7 +37,8 @@ Install ISV Operator From OperatorHub Via CLI    # robocop: disable
     Wait Until Keyword Succeeds    1 min    0 sec
     ...    Is Resource Present    Subscription    ${subscription_name}     ${namespace}    ${IS_PRESENT}
     Sleep    30s
```

Comment:
if we are waiting for the keyword to be successful, we dont need this sleep line

Reply:
removed sentence